src/App.jsx

```diff
@@ -6,23 +6,30 @@ import { Results } from "./pages/ResultsView/index.jsx";
 import { Venue } from "./pages/Venue/index.jsx";
 import { Layout } from "./components/Layout/index.jsx";
 import "./poppins.css";
+import React, { useState } from "react";
+
+export const Context = React.createContext();
```

Comment:
Context is a bad variable name. Ask yourself a question "What is the purpose of this context?" and put the answer as variable name

Reply:
done